File: FibonacciProblems/Fibonacci.java
package FibonacciProblems;
// This is a for loop approach to print 13 first fibonacci terms. 

public class Fibonacci{

    public static void main(String[] args) {
        int first = 0;
        int second = 1;

        for (int i=0; i<=12; i++) {
            int nextFiboNum = first + second;

            System.out.println(first);

            first = second;
            second = nextFiboNum;
        }
    }
}

File: FibonacciProblems/fibonacci.py
# // This is a for loop approach to print n first fibonacci terms. 

n = int(input("Enter a number of fibo terms: "))

previous_2nd = 0
previous_1st = 1

for i in range(n):
    print(previous_2nd, end=" ")
    next = previous_2nd + previous_1st
    previous_2nd = previous_1st
    previous_1st = next
